FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -26,14 +26,24 @@ class IssuesViewController : UITableViewController {
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        
-//        if let zip = UserDefaults.standard.string(forKey: UserDefaultsKeys.zipCode.rawValue) {
-//            
-//        }
+        NotificationCenter.default.addObserver(self, selector: #selector(zipCodeChanged(_:)), name: .zipCodeChanged, object: nil)
+        if let zip = UserDefaults.standard.string(forKey: UserDefaultsKeys.zipCode.rawValue) {
+            fetchIssues(forZip: zip)
```

Comment:
Ah I see you did it here. I don't think we need to reload issues every time the view appears, which gets called pretty frequently.  the data doesn't change so rapidly that we have to keep fresh data coming in as you switch back to the app, for instance.